app/manager/session_manager.js

```diff
@@ -0,0 +1,29 @@
+const jwt = require('jwt-simple'),
+  config = require('./../../config'),
+  User = require('./../models').user;
+
+const SECRET = config.common.session.secret;
+
+exports.decode = toDecode => {
+  return new Promise((resolve, reject) => {
+    let decodedToken;
+    try {
+      decodedToken = jwt.decode(toDecode, SECRET);
+    } catch (err) {
+      reject(err);
+    }
+    resolve(decodedToken);
+  });
+};
+
+exports.generateAccessToken = user => {
```

Comment:
This is just so you know an alternative, dont need to change it.
You can also write this as 
```
exports.generateAccessToken = user => ({
    token: jwt.encode(
      {
        authenticationCode: user.authenticationCode,
        id: user.id
      },
      SECRET
    )
 });
```